### ISSUE TITLE
Obsolete keyword argument in Himawari-8 True Color example

### ISSUE THREAD

**Issue by pnuu** [pytroll/pytroll-examples#48]
In [this](https://nbviewer.org/github/pytroll/pytroll-examples/blob/main/satpy/ahi_true_color_pyspectral.ipynb) example, `Scene` is given an obsolete keyword argument `sensor`. It appears the example was made when we still supported Python 2.7, so there might be some other obsolete bits, too.

**Comment by gerritholl** [comment]
Would it be possible to automatically run the notebooks for any PR in Satpy, Pyresample, and other relevant projects?

**Comment by pnuu** [comment]
Only if the data are available and there are enough resources to run with real data.

**Comment by gerritholl** [comment]
On the European Weather Cloud?  Then maybe not for every PR but overnight with main (if main has changed).

**Comment by mraspaud** [comment]
yes, we need to do that. For the benchmarking there, we already have some data locally too.

**Comment by djhoese** [comment]
I think if we had a cronjob on EWC that rsync'd (or at least compared what files exist locally before re-downloading) public ABI data from AWS, public AHI data from <somewhere>, and SEVIRI data from EWC then we could probably update the examples to use most of the same data. I also have my zip files of data from my tutorial that the various tutorials here could use and the EWC virtual machines could download/cache and then run notebooks against. It is just a matter of making the examples use relative paths.

I guess we could actually just make all of that done as part of the demo data subpackage in satpy. That's what it was meant for originally anyway. That way users don't have to wonder why Satpy is complaining about not having files that the examples are clearly expecting.
